### ISSUE TITLE
Cubemap always exports with exposure 4

### ISSUE THREAD

**Issue by HidekoHaruna** [matheowis/HDRI-to-CubeMap#26]
Please fix. It makes the image extremely bright.

Example:

_HDRI_
![image](https://github.com/user-attachments/assets/377d8230-c570-4b32-8335-b33e27d2c2e4)

_Cubemap_
![pz](https://github.com/user-attachments/assets/ce983f1f-3817-4c23-926d-c785c4e2a2e3)

**Comment by hello10293** [comment]
You probably tried this but did you lower the exposure at the bottom

![Image](https://github.com/user-attachments/assets/22f092f2-186b-44b4-b413-0983a100fc85)

?

**Comment by HidekoHaruna** [comment]
> You probably tried this but did you lower the exposure at the bottom
> 
> ![Image](https://github.com/user-attachments/assets/22f092f2-186b-44b4-b413-0983a100fc85)
> 
> ?

Yes, but the exported cubemap always looks as if the exposure was set to 4

**Comment by matheowis** [comment]
@HidekoHaruna  Can you share the HDRI that you have used? The first image is png, but you are importing HDRI to the app right?

**Comment by matheowis** [comment]
@HidekoHaruna You are right, unfortunately i left project years ago, and i don't have time to fix it, Sorry.
